kong/runloop/balancer/init.lua

```diff
@@ -97,6 +133,18 @@ local function get_value_to_hash(upstream, ctx)
         }
       end
 
+    elseif hash_on == "path" then
+      -- for the sake of simplicity, we're using the NGINX-normalized version of
+      -- the path here instead of running ngx.var.request_uri through our
+      -- internal normalization mechanism
+      identifier = var.uri
```

Comment:
just checking: `var.uri` is just the path right? it doesn't include the query args?